### PR TITLE
bugfix/daylight-end-catch

### PR DESCRIPTION
**Pull request by fivetran-joemarkiewicz**
## PR Overview
**This PR will address the following Issue/Feature:** #121 

**This PR will result in the following new package version:** `v0.13.1`

This will not be a breaking change. Instead, this will update the `int_zendesk_schedule_spine` model so schedules that have daylight saving times that have ended are properly being reported into the future. As none of the downstream models are incremental, this will not be a breaking change.

**Please provide the finalized CHANGELOG entry which details the relevant changes included in this PR:**

### Basic Validation
Please acknowledge that you have successfully performed the following commands locally:
- [X] dbt run –full-refresh && dbt test
- [X] dbt run (if incremental models are present) && dbt test

Before marking this PR as "ready for review" the following have been applied:
- [x] The appropriate issue has been linked, tagged, and properly assigned
- [x] All necessary documentation and version upgrades have been applied
- [X] docs were regenerated (unless this PR does not include any code or yml updates)
- [x] BuildKite integration tests are passing
- [X] Detailed validation steps have been provided below

### Detailed Validation
Please share any and all of your validation steps:

Please inspect the corresponding Height ticket for detailed validations of these changes. The validations for this PR include the following checks:
- Understanding of the schedule (in this example Hong Kong)
- Understanding of how the Hong Kong schedule behaves when DST ends at a specified point in time.
- Validation of how the changes in this PR fix the issue for a ticket using the Hong Kong schedule in the current day and age when DST is no longer used.

### If you had to summarize this PR in an emoji, which would it be?
 
⌛

### CODE REVIEW

File: .github/workflows/auto-release.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+    types:
+      - closed
+    branches:
+      - main
+
+jobs:
+  call-workflow-passing-data:
+    if: github.event.pull_request.merged
+    uses: fivetran/dbt_package_automations/.github/workflows/auto-release.yml@main
+    secrets: inherit
```
Comment (fivetran-jamie) [inline .github/workflows/auto-release.yml:13]: just added the secret

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,15 @@
+# dbt_zendesk v0.14.0
+
+## Bug Fixes
```
Comment (fivetran-jamie) [inline CHANGELOG.md:3]: should we note that this only affects people who use schedules in zendesk? might be implied bust just to be uber clear

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,15 @@
...
+- The `int_zendesk__schedule_spine` model was updated to properly account for schedules that recognized daylight savings time (DST) at one point in time, and then stopped recognizing it at a later date. ([PR #128](https://github.com/fivetran/dbt_zendesk/pull/128))
+  - For example, the Hong Kong timezone originally recognized DST, but them stopped in 1979. The previous versions of this package only recorded the schedule business hours until 1979. This update addresses this bug.
+
+## Under the Hood
+- Included auto-releaser GitHub Actions workflow to automate future releases. ([PR #128](https://github.com/fivetran/dbt_zendesk/pull/128))
```
Comment (fivetran-jamie) [inline CHANGELOG.md:8]: should we include the updated maintainer PR template? or is it too under the hood...

**Review by fivetran-jamie — APPROVED** [review]
approved with some doc-related suggestions! also reminder to regen docs

File: CHANGELOG.md (hunk 3 above)
Comment (fivetran-joemarkiewicz) [reply CHANGELOG.md:8]: I think that is a good think to include. Will update and thanks for calling out.

File: CHANGELOG.md (hunk 2 above)
Comment (fivetran-joemarkiewicz) [reply CHANGELOG.md:3]: Yeah that's a good clarification to include.

File: README.md [hunk 4]
```diff
@@ -59,7 +59,7 @@ Include the following zendesk package version in your `packages.yml` file:
 ```yml
 packages:
   - package: fivetran/zendesk
-    version: [">=0.13.0", "<0.14.0"]
+    version: [">=0.14.0", "<0.15.0"]
```
[the remark this answers is not in the document]
Comment (fivetran-joemarkiewicz) [reply README.md:62]: Yeah I had initially marked this as a breaking change as I was planning to include this in the next upcoming breaking change. But if we are going to introduce this earlier as a patch then we should adjust the versioning throughout.

File: CHANGELOG.md (hunk 3 above)
Comment (fivetran-joemarkiewicz) [reply CHANGELOG.md]: Updated.

File: CHANGELOG.md (hunk 2 above)
Comment (fivetran-joemarkiewicz) [reply CHANGELOG.md:4]: Updated.

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,3 +1,18 @@
+# dbt_zendesk v0.13.1
+
+[PR #128](https://github.com/fivetran/dbt_zendesk/pull/128) includes the following changes:
+## Bug Fixes
+- The `int_zendesk__schedule_spine` model was updated to properly account for schedules that recognized daylight savings time (DST) at one point in time, and then stopped recognizing it at a later date.
+  - For example, the Hong Kong timezone originally recognized DST, but them stopped in 1979. The previous versions of this package only recorded the schedule business hours until 1979. This update addresses this bug.
```
Comment (fivetran-catfritz) [inline CHANGELOG.md:6]: Just a minor fix:
```suggestion
  - For example, the Hong Kong timezone originally recognized DST, but then stopped in 1979. The previous versions of this package only recorded the schedule business hours until 1979. This update addresses this bug.
```

**Review by fivetran-catfritz — APPROVED** [review]
approved for release.

**Pull request merged**